### ISSUE TITLE
The cells are not scrolling correctly

### ISSUE THREAD

**Issue by W7orld** [evrencoskun/TableView#311]
The cells are not scrolling correctly.. Some of them not scrolled with others.. How can i fixed?

Check this images
![1](https://user-images.githubusercontent.com/50217561/79729470-9322ee00-8300-11ea-88b5-f5386ca8c886.jpg)

![2](https://user-images.githubusercontent.com/50217561/79729498-9c13bf80-8300-11ea-8528-e880594f0b1d.jpg)

**Comment by fiterzs** [comment]
@W7orld  HI,Have you solved this problem?Im also have this problem

**Comment by W7orld** [comment]
> @W7orld HI,Have you solved this problem?Im also have this problem

Hi, Not exactly but i used nestedscrollview and it's fine now it's works better
I putted the TableView inside Nestedscrollview

**Comment by swatikch** [comment]
I have a similar issue. When I try using Nestedscrollview, the columns disappear. Did you come across similar issue @W7orld ?

**Comment by W7orld** [comment]
@swatikch No.  Did you have other ViewObject inside the Nestedscrollview I mean with TableView dose he only inside Nestedscrollview?

**Comment by haroutj** [comment]
@swatikch I also had this issue. I fixed it by making a separate xml file for the TableView and then inflating the TableView and adding it as a child of the NestedScrollView at runtime. Let me know if you have any problems.
